tests/behat/features/capabilities/group/group-create-secret.feature

```diff
@@ -0,0 +1,182 @@
+@api @group @stability @stability-1 @group-create-secret
+Feature: Create Secret Group
+  Benefit: I want to create a secret group, where only group members can see the content.
+  Role: As a LU
+  Goal/desire: I want to create Secret Groups
+
+  Scenario: Successfully create secret group
+    Given users:
+      | name             | mail                     | status | roles       |
+      |  SecretGroup User One   | group_user_1@example.com | 1      | sitemanager |
+      |  SecretGroup User Two   | group_user_2@example.com | 1      |             |
+    And I enable the module "social_group_secret"
+    And I am logged in as an "authenticated user"
+    And I am on "user"
+    And I click "Groups"
+    And I click "Add a group"
+    Then I should not see "Secret group"
```

Comment:
I think we should split everything up to here into a separate Scenario `As a normal user I can not create a secret group`